tests/timing-for-ci.txt

```diff
@@ -1,1020 +1,1038 @@
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/03/03/issue-4621.qmd
+        7.60 real 7.22 user 0.71 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/03/28/remote-resources.qmd
+        3.55 real 3.99 user 0.48 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/03/01/issue-4568.qmd
+        6.83 real 6.33 user 0.71 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/03/01/issue-4586.qmd
+        3.59 real 4.13 user 0.54 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/11/unicode-subref-labels.qmd
+        8.07 real 7.65 user 0.79 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/06/theorem-ast.qmd
+        3.42 real 3.91 user 0.44 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/02/6568.qmd
+        3.60 real 4.01 user 0.50 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/02/6853.qmd
+        3.15 real 3.59 user 0.43 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/02/7108.qmd
+        2.66 real 3.15 user 0.34 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/04/5637.qmd
+        3.65 real 4.07 user 0.45 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/04/6077.qmd
+        2.69 real 3.10 user 0.36 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/04/content-hidden.qmd
+        3.89 real 4.35 user 0.43 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/04/6965.qmd
+        2.77 real 3.21 user 0.38 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/03/7118.qmd
+        7.61 real 7.14 user 0.73 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/16/7236.qmd
+        8.01 real 8.24 user 0.55 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/12/7215.qmd
+        3.48 real 3.98 user 0.39 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/10/issue-7187-b.qmd
+        3.45 real 3.94 user 0.40 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/10/issue-7187-jupyter.qmd
+        8.76 real 8.33 user 1.15 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/10/issue-7187.qmd
+        3.33 real 3.91 user 0.32 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/10/10/issue-7187-knitr.qmd
+        7.30 real 6.86 user 0.79 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/01/13/lua-raw-html.qmd
+        3.49 real 3.96 user 0.39 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/01/20/test-quarto-disable-processing.qmd
+        3.48 real 3.91 user 0.43 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/01/20/asciidoc.qmd
+        2.64 real 3.01 user 0.31 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/01/23/reveal-config-quote-4063.qmd
+        2.78 real 3.24 user 0.37 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/01/23/table-options.qmd
+        3.41 real 3.86 user 0.41 sys
+./smoke/smoke-all.test.ts -- docs/smoke-all/2023/01/18/_file
+        2.07 real 2.61 user 0.20 sys
+./smoke/smoke-all.test.ts -- with
+        2.05 real 2.39 user 0.22 sys
+./smoke/smoke-all.test.ts -- spaces.md
+        2.10 real 2.51 user 0.23 sys
```

Comment:
This is to fix. File with `_` should be ignored

And files with spaces are an issue.

Reply:
Resolved by dcae8ce2bc920a94f74effdc7507db242371e450